test/integration/integration_suite_test.go

```diff
@@ -79,7 +80,8 @@ func init() {
 	flag.StringVar(&manifest, "manifest-path", "./config/nephe.yml", "The relative path to manifest.")
 	flag.BoolVar(&preserveSetupOnFail, "preserve-setup-on-fail", false, "Preserve the setup if a test failed.")
 	flag.StringVar(&supportBundleDir, "support-bundle-dir", "", "Support bundles are saved in this dir when specified.")
-	flag.BoolVar(&withAgent, "with-agent", false, "Using antrea-agent on VM.")
+	flag.BoolVar(&withAgent, "with-agent", false, "Using antrea-agent on Linux VM.")
+	flag.BoolVar(&withWindows, "with-windows", false, "Using antrea-agent on Windows VM")
```

Comment:
Both `with-agent` and `with-windows` need to be set for windows agented ci to work right? What would happen if only `with-windows` is set but not with-agent? I see there are flags throughout the PR, so it's not just this case.

Reply:
Added validation to verify that when -with-windows is true, -with-agent should be true as well, otherwise the test will throw an error